### DISCUSSION TITLE
How to disable debug logs

### DISCUSSION THREAD

**Discussion by dmytro-i** [oracle/dotnet-db-samples#316] [Q&A] [answered]
Hello!

My `main` looks like this

```            IHost host = Host.CreateDefaultBuilder(args)
                .ConfigureServices(services =>
                {
                    services.AddOracle<MyDataContext>($"name={nameof(MyDataContext)}")
                        .AddHostedService<MyService>();
                })
                .ConfigureLogging(logger =>
                {
                    logger.ClearProviders();
                    logger.AddConsole();
                })
                .Build();

            await host.RunAsync();
```

And my appsettings.json like this

```{
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "Microsoft": "Warning"
    }
  }
}
```
When I run my app on the server I see queries being logged. They don't have a log level or logger so I can't filter them out using `appsettings.json`. It only happens on my Ubuntu server and not on my Mac.
I'm running the app with `.net core 7.0`
[This documentation page](https://docs.oracle.com/en/database/oracle/oracle-data-access-components/19.3/odpnt/EFCoreLogging.html) shows how to setup logging but I don't have anything like this in my code.

**Comment by dmytro-i** [comment]
<img width="1490" alt="image" src="https://github.com/oracle/dotnet-db-samples/assets/12176944/1afcd33b-5c69-442d-8df6-55c1a76a6e66">
Logs look like this

**Comment by alexkeh** [comment, marked as answer]
Are you using EF Core 7? There is a known Oracle EF Core bug (35579522) that is fixed in Oracle EF Core 7.21.12. It eliminates previously reported Debug.WriteLine calls. These were debug calls were not present with EFC 6. The patch is planned to be published on NuGet this month.

**Reply by dmytro-i** [reply]
Yes, it's EF Core 7. Glad to hear that the fix is coming. Thanks again for taking the time to answer questions.

**Reply by dmytro-i** [reply]
I confirm that it is fixed in `7.21.12`
